src/routes/(auth)/login/+server.ts

```diff
@@ -51,6 +51,7 @@ export const POST = async ({
     if (!username || !password) {
       return new Response(
         JSON.stringify({
+          error: 'missing_credentials',
```

Comment:
Please create an enum and use it on uppercase to follow same conventions as for GraphQL error codes in our schema.

```ts
export enum LoginError {
  MissingCredentials = 'MISSING_CREDENTIALS',
}
```

Then here we could do


```suggestion
          error: LoginError.MissingCredentials
```

You can import `LoginError` in the front-end code and use it directly for the response.